package/rdspgbadger.py

```diff
@@ -126,15 +133,29 @@ def write_log(client, dbinstance_id, filename, logfilename):
                     raise
         with open(filename, "a") as logfile:
             if 'LogFileData' in response:
-                logfile.write(response["LogFileData"])
-
-        if not response["AdditionalDataPending"]:
+                if truncated_string in response["LogFileData"][-slice_length:]:
+                    downloaded_lines = response["LogFileData"].count("\n")
+                    if downloaded_lines == 0:
+                        raise Exception(
+                            "No line was downloaded in last portion!")
+                    max_number_of_lines = max(downloaded_lines - 10, 1)
```

Comment:
I would just create a variable with this arbitrary number at the same level of `marker` and `max_number_of_lines`

Reply:
Makes perfect sense 👍